diff --git a/lib/parse-url.js b/lib/parse-url.js
index 7d5489c008ab4a0a57cbf31043557ae6e1082f92..82efa6af5eecf8e3a5688087161b7fa6822031fb 100644
--- a/lib/parse-url.js
+++ b/lib/parse-url.js
@@ -1,4 +1,4 @@
-const url = require('url')
+// const url = require('url')
 
 const lastIndexOfBefore = (str, char, beforeChar) => {
   const startPosition = str.indexOf(beforeChar)
@@ -7,7 +7,7 @@ const lastIndexOfBefore = (str, char, beforeChar) => {
 
 const safeUrl = (u) => {
   try {
-    return new url.URL(u)
+    return new URL(u)
   } catch {
     // this fn should never throw
   }
